content/authorization/sign-in-with-livechat/index.mdx

```diff
@@ -66,268 +77,499 @@ The SDK is available in two formats: an NPM module and a CDN-hosted library.
+
+</Warning>
+
+# SDK documentation
+
+## Classes
+
+### **AccountsSDK**
+
+The main instance of the SDK used to authorize users in LiveChat Accounts.
+
+#### **constructor(options = {})**
+
+<CodeSample path={'Example class creation'}>
+
+```js
+const instance = new AccountsSDK({
+  client_id: '<your_app_client_id>'
+});
+```
+
+</CodeSample>
+
+##### options object
+
+The constructor accepts an `options` object with the following properties:
+
+| Property                             | Required | Description                                                                                                                                                  |
+| ------------------------------------ | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `client_id`                          | yes      | Application **Client Id** (can be found in <a href="https://developers.livechat.com/console/" rel="noopener nofollow" target="_blank">Developer Console</a>) |
+| `prompt`                             | no       | Use `consent` to force the consent prompt in a popup and the redirect flows. ([See the warning.](#step-3-prepare-button-container)). Default: `''`           |
+| `response_type`                      | no       | OAuth response type. Possible values: `token` or `code`. Default: `token`                                                                                    |
+| `popup_flow`                         | no       | `auto` - close the popup when not required; `manual` - always show the popup; default: `auto`                                                                |
+| `state`                              | no       | OAuth state parameter. Use it for better security.                                                                                                           |
+| `verify_state`                       | no       | Check if `state` matches after the redirect; default: `true`                                                                                                 |
+| `scope`                              | no       | Request exact scopes - must be configured for a given `client id`, a comma-separated string, default: `null`                                                 |
+| `redirect_uri`                       | yes      | OAuth redirect URI; default: the current location                                                                                                            |
+| `email_hint`                         | no       | Fill in email in forms                                                                                                                                       |
```

Comment:
Don't we need to mention default values for these params?

Reply:
I think it would be nice, gonna add it in a moment

Reply:
Hmmm, there are no default params for this properties, if they aren't provided then the ystays null/undefined